person.rb

```diff
@@ -3,14 +3,15 @@
 # Represents a person with basic information such as name, age, and permission to use services.
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
```

Comment:
- Hey, the setter/getter method should include the books and rentals so their values could be accessible. let's improve on it like this. `attr_accessor :name, :age, :books, :rentals`